assets/app/view/game/alternate_corporations.rb

```diff
@@ -0,0 +1,257 @@
+# frozen_string_literal: true
+
+module View
+  module Game
+    module AlternateCorporations
```

Comment:
I feel like calling this "AlternateCorporations" is too general. This implementation seems very specific to the mining corps.

Reply:
My thinking was that other titles may have similar needs and would be able to put the new views in this file. I didn't want anything in the UI::Corporation code to reference mines.